addons/mrp/tests/test_warehouse_multistep_manufacturing.py

```diff
@@ -29,7 +29,6 @@ def setUpClass(cls):
         product_form = Form(cls.env['product.product'])
         product_form.name = 'Stick'
         product_form.uom_id = cls.uom_unit
-        product_form.uom_po_id = cls.uom_unit
```

Comment:
but why?

Reply:
for some reason, idk, the warning `you can't write on an invisible field`